spec/cypress/integration/Cookies.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Cookie page
+  Everybody wants to set cookie preferences
+
+  Scenario: Setting cookie preferences on cookie page
+    Given I am on "cookie" page
+    And "cookie consent" radios are unchecked
+
+    When I set "cookie consent" radio to "on"
+    And I click the submit button
+    Then cookie preferences have changed
+    And "cookie consent" radio with value "on" is checked
+
+    When I set "cookie consent" radio to "off"
+    And I click the submit button
+    Then cookie preferences have changed
+    And "cookie consent" radio with value "off" is checked
```

Comment:
So my 'pipe dream' was something like
```
  Scenario: Setting cookie preferences on cookie page
    Given I am on "cookie" page
    And "cookie consent" radios are unchecked
      When I set "cookie consent" radio to "on"
      And I click the submit button
        Then cookie preferences have changed
        And "cookie consent" radio with value "on" is checked
      When I set "cookie consent" radio to "off"
      And I click the submit button
        Then cookie preferences have changed
        And "cookie consent" radio with value "off" is checked
```
So you give more indentation to things that come further in the sentence and are more specific. 

But I think cucumber might not think it's a good syntax :P

Reply:
I feel like decent syntax highlighting helps with this a lot!

![image](https://user-images.githubusercontent.com/472830/111614898-0d414880-87d8-11eb-9ba7-f7924b77026e.png)

Reply:
Yeah, locally.

I am not sure there is an easy way of highlighting syntax in github PRs though :P

Reply:
I'm against indentation purely because they are supposed to be kind of read like English, and indenting them like they're code seems to go against that to me

It does seem to allow it though

Reply:
Hmm - not sure if I like indentation - it actually takes more time to understand it. It 'doesnt feel right' for lack of etter expression. @amazimbe @ethanmills - What are your opinions about indentation?

Reply:
No strong opinions either way. For me, the highlighted syntax is definitely easiest to read.